tunedev/Banka-react
# #167122089 setup react tools

#### What does this PR do?
It sets up the basic react configuration

#### Description of Task to be completed?
> - add webpack configuration and tools needed for building project in react
> - add babel package and its config file
> - add eslint and prettier config
> - set up jest and react testing library
> - add @reach/router

#### How should this be manually tested?
> - Clone this branch
> - Run `npm install` and `npm run dev`
> - A page that indicates that the setup working should open in your default browser

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#167122089](https://www.pivotaltracker.com/story/show/167122089)

#### Screenshots (if appropriate)
NA

#### Questions:
NA
